catalogue/webapp/components/ExpandedImage/ExpandedImage.js

```diff
@@ -133,12 +192,25 @@ const ExpandedImage = ({ title, index, id, searchParams }: Props) => {
     fetchDetailedWork();
   }, []);
 
+  useEffect(() => {
+    document &&
+      document.documentElement &&
+      document.documentElement.classList.add('is-scroll-locked');
+
+    return () => {
+      document &&
+        document.documentElement &&
+        document.documentElement.classList.remove('is-scroll-locked');
+    };
+  }, []);
+
```

Comment:
As several of the above useEffects all get called when the component mounts, could they be bundled together into a single useEffect ?

Reply:
They could. [My take was that React encourages you to split different concerns into different `useEffect`s](https://reactjs.org/docs/hooks-effect.html#tip-use-multiple-effects-to-separate-concerns), which can improve readability and refactoring. I suppose there could be a perf penalty though.

Reply:
👍

Reply:
Yea, the hooks implementation deals with this if you fancy digging into it.